docs/development/chromium-development.md

```diff
@@ -2,7 +2,7 @@
 
 > A collection of resources for learning about Chromium and tracking its development
 
-- [chromiumdev](https://chromiumdev-slack.herokuapp.com) on Slack
+- [#chromium](irc://irc.freenode.net/chromium) on Freenode IRC
```

Comment:
Our linter doesn't like the irc:// protocol.  Can we use a different link or remove the link?